.gitlab/kernel_matrix_testing/security_agent.yml

```diff
@@ -143,12 +143,12 @@ kmt_run_secagent_tests_x64:
   parallel:
     matrix:
       - TAG:
-          - "ubuntu_18.04"
-          - "ubuntu_20.04"
-          - "ubuntu_22.04"
-          - "ubuntu_23.10"
```

Comment:
I don't understand this change, ubuntu releases 2 versions per year 2x.04 and 2x.10, how do you differentiate between the 2 ?

Reply:
With this PR we switch to using the information directly from the ami-builder repository, including the distro names. There, the Ubuntu versions are just defined by the release year (we're always using LTS except for 23.10, which is not a LTS year). I agree that it can be confusing though, so I'll make a followup PR in that repo and change the names.

Reply:
Before, we were using custom names in the `platform.json` as that was edited manually. Now we switch to automatic generation, so we use the same names there are in the ami-builder repo. There, the Ubuntu versions are defined just with the release year. As we only have one image per release year there's no risk of using different images in the CI. However I agree that it can be confusing for the devs, so I'll make a followup PR in ami-builder to use unambigous names and then update them here.

Reply:
Honestly I still disagree, in december of 2024 we will most likely want to have both 24.04 (the latest LTS) and 24.10 (the latest ubuntu, playing the role of 23.10 currently), I don't see how you represent that here

Reply:
So yeah to me the real issue is https://github.com/DataDog/ami-builder/blame/7cfb8eda153a86bf46d70bd98908639342927529/.gitlab/distro.yml#L105, which really does not make any sense

Reply:
Indeed, I'll make a PR changing those names and then update them here.